src/OpenSage.Game/Content/SubsystemLoader.cs

```diff
@@ -29,7 +29,8 @@ public void Load(Subsystem subsystem)
                     LoadFiles(
                         @"Data\INI\Default\GameData.ini",
                         @"Data\INI\GameData.ini",
-                        @"Data\INI\Mouse.ini");
+                        @"Data\INI\Mouse.ini",
+                        @"Maps\MapCache.ini");
```

Comment:
Required to load official maps. Not sure if the Core subsystem is the correct place

Reply:
I think we'll want a `Subsystem.MapCache`, but I don't mind if that is done now or later. We might even want `MapCacheSinglePlayer` and `MapCacheMultiPlayer`, because that's C&C3 stores those things separately.